### PR TITLE
KHWE-81: Connect Next/Previous buttons with songs

### PR DESCRIPTION
**Pull request by pavelbaranchuk**

### CODE REVIEW

File: src/components/albums/AlbumTable.js [hunk 1]
```diff
@@ -90,7 +90,10 @@ class AlbumTable extends Component {
                     </Typography>
                   </div>
                 </div>
-                <TableLayout songs={this.props.songs} />
+                <TableLayout
+                  songs={this.props.songs}
+                  match={this.props.match.params.id}
```
Comment (ihor4568) [inline src/components/albums/AlbumTable.js]: why is this prop called match if it is id?

Comment (ihor4568) [reply src/components/albums/AlbumTable.js]: mb currentSongId?

File: src/components/mySongs/MySongs.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ class MySongs extends Component {
 
     return (
       <>
-        <Title name="My Songs" />
+        <Title name={"My Songs"} />
```
Comment (ihor4568) [inline src/components/mySongs/MySongs.js]: why did you need to change this?

File: src/store/actionCreators/player.js [hunk 3]
```diff
@@ -1,16 +1,25 @@
...
-    type: PLAY_SONG,
-    song
+    type: actionTypes.PLAY_SONG,
+    song,
+    number
   };
 }
 
-export function pauseSong(song) {
+export function pauseSong(song, number) {
```
Comment (pavelbaranchuk) [inline src/store/actionCreators/player.js]: please check

File: src/components/player/PlayerContainer.js [hunk 4]
```diff
@@ -91,12 +91,40 @@ export class PlayerContainer extends Component {
...
+
+  handleNextSong = () => {
+    const { pauseSong, playSong } = this.props;
+    const { isPlaying, number, savedSongs } = this.props.player;
+
+    if (isPlaying) {
+      pauseSong(savedSongs[number], number);
+    }
+
+    number + 1 === savedSongs.length
```
Comment (pavelbaranchuk) [inline src/components/player/PlayerContainer.js]: please rewrite
